scripts/import_data.py

```diff
@@ -0,0 +1,480 @@
+#!/usr/bin/env python3
+
+import argparse
+import ast
+import datetime
+import ipaddress
+import json
+import random
+import re
+import subprocess
+import sys
+import uuid
+
+try:
+    import mysql.connector
+    from mysql.connector.connection import MySQLCursor
+    import pandas as pd
+    import yaml
+    import xlrd  # used by pandas
+except ImportError:
+    print("The following modules are required:")
+    print("mysql-connector-python")
+    print("pandas")
+    print("PyYAML")
+    print("xlrd")
+    print("You can install them with: pip install {module name}")
```

Comment:
We print print exc here and user will know which module is missing.

Please consider